docker-compose.yml

```diff
@@ -43,7 +44,7 @@ services:
         if [ "${STORE_API_ENABLED:-yes}" == "no" ]; then
           exit 0
         fi
-        INIT_QUEPASA=false INIT_PEPPERMINT=false ./script/migrate up
+        INIT_QUEPASA=false INIT_PEPPERMINT=false ./script/migrate
```

Comment:
Shouldn't we at this point be calling the `script/entrypoint`? That is, I think the entrypoint in the docker-compose.yml here should be defined as follows:

```
    entrypoint: |-
      bash -c '
        set -e
        if [ "${STORE_API_ENABLED:-yes}" == "no" ]; then
          exit 0
        fi

        ./script/entrypoint    entrypoint: |-
      bash -c '
        set -e
        if [ "${STORE_API_ENABLED:-yes}" == "no" ]; then
          exit 0
        fi
      '
```

And similar in the admin-api entrypoint.

Reply:
Both are fixed now